core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1555,6 +1559,9 @@ class SparkContext(config: SparkConf) extends Logging {
       Utils.fetchFile(uri.toString, new File(SparkFiles.getRootDirectory()), conf,
         env.securityManager, hadoopConfiguration, timestamp, useCache = false)
       postEnvironmentUpdate()
+    } else {
+      logWarning(s"The path $path has been added already. Overwriting of added paths " +
```

Comment:
Shell we leave a JIRA link as a comment for example SPARK-16787 and/or SPARK-19417

Reply:
We normally do not post the JIRA number in the message.

Reply:
I meant a comment.